database/mssql/src/test/java/org/ballerinalang/scenario/test/database/mssql/CallTest.java

```diff
@@ -49,6 +49,13 @@
     private String userName;
     private String password;
     private Path resourcePath;
+    private static long date;
+    private static long time;
+    private static long datetime;
+    private static long datetime2;
```

Comment:
Shall we use a more meaningful variable name here?

Reply:
Here datetime2 is a data type in sql server.
http://www.sqlservertutorial.net/sql-server-basics/sql-server-datetime2/